lib/lz4.c

```diff
@@ -1725,7 +1725,16 @@ LZ4_decompress_generic(
 
                 /* Fastpath check: Avoids a branch in LZ4_wildCopy32 if true */
                 if (!(dict == usingExtDict) || (match >= lowPrefix)) {
+#if defined(__aarch64__) && defined(__GNUC__)
```

Comment:
This change will also be enabled for `clang`, since it also defines `__GNUC__`. 
Is that the intention ?

Reply:
Currently, FAST_DEC_LOOP only enabled on AARCH64 with GCC.
So I will change like this
#if defined(__aarch64__) && defined(__GNUC__) && !defined(__clang__)

Reply:
yes

Reply:
@Cyan4973 
Fixed.
Please review latest commit